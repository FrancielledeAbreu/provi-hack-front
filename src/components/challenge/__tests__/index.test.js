import React from "react";
import CardChallenge from "../index";
import renderer from "react-test-renderer";

describe("CardChallenge Test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <CardChallenge
          name="Front-end"
          challenges_type="single"
          image="https://image.freepik.com/fotos-gratis/ilustracao-da-palavra-da-imagem_93826-815.jpg"
          level="medium"
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
