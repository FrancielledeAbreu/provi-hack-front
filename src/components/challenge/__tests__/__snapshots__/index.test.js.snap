// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardChallenge Test renders correctly 1`] = `
<div
  className="sc-gsTCUz gdfIMh"
>
  <img
    alt=""
    className="sc-bdfBwQ hcwlGI"
    src="https://image.freepik.com/fotos-gratis/ilustracao-da-palavra-da-imagem_93826-815.jpg"
  />
  <img
    alt=""
    className="sc-bdfBwQ hcwlGI"
    src="medium.png"
  />
  <span>
    Front-end
  </span>
  <img
    alt=""
    className="sc-bdfBwQ hcwlGI"
    src="single.png"
  />
</div>
`;
